cloudera/assets/configuration/spec.yaml

```diff
@@ -41,4 +41,21 @@ files:
       value:
         type: integer
         example: 100
+    - name: clusters
+      description: |
+        Autodiscover clusters config.
```

Comment:
Can you include a detailed description and examples of how this configuration is expected to be used?

Reply:
What do you think about updating the README with that in a future PR?

Reply:
After speaking with Andrew, we are going to include it in this PR